gui/themes/common/lowres_layout.stx

```diff
@@ -1266,6 +1266,12 @@
 	</dialog>
 
 	<dialog name = 'GameOptions_Game' overlays = 'Dialog.GameOptions.TabWidget' shading = 'dim'>
+		<layout type = 'vertical' padding = '0, 0, 0, 0' spacing = '0'>
+			<widget name = 'Container'/>
+		</layout>
+	</dialog>
+	
+	<dialog name = 'GameOptions_Game_Container' overlays ='Dialog.GameOptions_Game.Container' inset='0'>
 		<layout type = 'vertical' padding = '8, 8, 8, 8'>
 			<layout type = 'horizontal' padding = '0, 0, 0, 0' spacing = '6' align = 'center'>
```

Comment:
now that we have a scrollbar, we could remove these horizontal layouts from lowres if we wanted to